modules/auth/auth.constants.js

```diff
@@ -0,0 +1,130 @@
+// Package imports
+const env = require('dotenv').config();
+
+module.exports = {
+  responseMessages: {
+    authorizedUser: {
+      failure: 'You are not authorized. Please login first',
+    },
+    logOutUser: {
+      success: 'You are logged out successfully.',
+      failure: 'Failed logging out. Please try again!',
+    },
+    changePassword: {
+      wrongCredentails: 'Wrong Crendentials. Please try again!',
+      conflict: 'You are signed in through google/github.',
+      success: 'Your password is changed successfully',
+      failure: 'Failed to change password. Please login again and then try!',
+    },
+    generalErrorMessage: 'Something went wrong! Try again!',
+    forgotPassword: {
+      resetPassword: {
+        success: 'Your password has been reset successfully',
+        failure: 'Failed to reset password. Please try again!',
+        tokenExpiredMessage: 'Your token has expired. Please try again',
+        wrongCredentails: 'Wrong Crendentials. Please try again!',
+      },
+      resetPasswordEmail: {
+        success: 'Password reset mail is sent to your account',
+        failure: 'Password reset mail could not be sent! Check your network connectivity!',
+        emailText: `Please click on the link to reset your password. The link is active for 5 minuts only!`,
+        emailSubject: 'Reset you password!',
+      },
+    },
+    userProfileData: {
+      success: 'Your data has been fetched successfully',
+      failure: 'Failed to fetch data. Please try again!',
+    },
+    updateUserProfile: {
+      success: 'Your data has been updated successfully',
+      failure: 'Failed to update data. Please try again!',
+    },
+  },
+  labels: {
+    session: 'session: ',
+    sessiodId: 'sessionId: ',
+    user: 'user: ',
+    error: 'error: ',
+    info: 'info: ',
+    middleware: 'middleware: ',
+    loginCtrl: {
+      failure: 'user is not found in request. login() problem. (loginCtrl)',
+      success: 'User is loggedin',
+    },
+    signupCtrl: {
+      success: 'User is signed ip',
+      failure: 'user is not found in request. login() problem. (signupCtrl)',
+    },
+    logoutCtrl: {
+      failure: 'Error while logging out the user (logoutCtrl)',
+      success: 'User is loggedout',
+    },
+    changePasswordCtrl: {
+      noUserInReq: 'user is not found in request. login() problem. (changePasswrodCtrl)',
+      findingUser: {
+        failure: 'Error while finding user (changePasswordCtrl)',
+      },
+      savingNewPassword: {
+        failure: 'Error while saving new password (changePasswordCtrl)',
+        success: 'saved new password in db',
+        savingMsg: 'Saving new password in db',
+      },
+    },
+    forgotPasswordCtrl: {
+      findingUser: {
+        failure: 'Error while finding user (forgotPasswordCtrl)',
+      },
```

Comment:
Why response messages are added in the labels field?

Reply:
Issue is resolved